crates/swc_css_parser/tests/fixture/turbo/4092/input.css

```diff
@@ -0,0 +1,4 @@
+.carousel .control-dots .dot.selected, .carousel .control-dots .dot:hover {
+     opacity: 1;
+     filter: alpha(opacity=100);
```

Comment:
That is also deprecated syntax, I think we should move it under the option like https://github.com/swc-project/swc/pull/7110

Reply:
But I am fine with it because we have `PreservedToken`, we can handle all non standard cases late and move it under the option

Reply:
Can you recommend a name for the option?
Maybe `legacy_ie`?